package.json

```diff
@@ -10,8 +10,9 @@
     "email": "info@ddem.us"
   },
   "main": "index.js",
-  "engine": {
-    "node": "12.14.1"
+  "engines": {
+    "node": "^14.17.1",
+    "npm": "^6.13.4"
```

Comment:
wasn't sure if the `npm` version should be specified but figured it'd be helpful given that v7 is generally available

Reply:
Good idea. We have not got around to moving to npm v7 yet. It's been quite annoying, because npm v7 updates all the lock files, which happened to me when my npm_merge_driver was using v7 while the project was using v6. Let's discuss package managers at some point, e.g. pnpm vs. npm vs yarn